### PR TITLE
fabric vms provision

### PR DESCRIPTION
**Pull request by jdsheehan**
Ansible based example, for provisioning Hyperledger Fabric native binaries across multiple vms.

### CODE REVIEW

File: labs/fabric-vms-provision.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+Using the template of the first example, the second example creates a fully distributed Fabric network.
+Since this would be an extremely error prone process if done manually, an ansible playbook is provided instead, addressing the numerous roles required for the network.
+The Fabric conponents run on seperate vms, which are provided by softlayer.
+
+# Initial Committers
+- https://github.com/jdsheehan
+- https://github.com/eragnoli
+
+# Sponsor
+- https://github.com/christo4ferris  Christopher Ferris (chrisfer@us.ibm.com) - Role ("Member of Hyperledger board")
```
Comment (tkuhrt) [inline labs/fabric-vms-provision.md]: "board" --> "TSC"

File: labs/fabric-vms-provision.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Lab Name
+fabric-vms-provision
+
+# Short Description
+This entry will provide an example on how to provision, using ansible, hyperledger fabric native binaries on multiple softlayer vms. A build your second network example.
```
Comment (tkuhrt) [inline labs/fabric-vms-provision.md]: Can we capitalize Ansible and Hyperledger Fabric?

**Comment by tkuhrt** [comment]
Hi, @jdsheehan. 

Thanks for proposing this Lab. I have been asked many times for a way of creating a distributed Hyperledger Fabric network. So this will be extremely useful. I am curious as to why not just add these directly to the https://github.com/hyperledger/fabric-samples repo? Will that happen eventually?

**Comment by jdsheehan** [comment]
Hi @tkuhrt,

Christopher Ferris suggested here, but if there is interest I don't mind it eventually migrating to fabric-samples.

**Comment by lehors** [comment]
LGTM +1

**Comment by trbs** [comment]
LGTM +1

**Pull request merged**